### DISCUSSION TITLE
Add informative footnote to plots

### DISCUSSION THREAD

**Discussion by ruoranepi** [epiforecasts/EpiNow2#633] [Ideas]
For the standard 3 panel plot() from epinow - it would be very informative to add footnotes that explicitly state some obvious assumptions to avoid people misinterpreting the output. e.g., a footnote on "Rt is assumed to be constant during the forecasting period".

**Comment by sbfnk** [comment]
This is a great suggestion - the only problem I see is that while Rt being assumed to be constant is the default this can be changed, see e.g. the example at https://epiforecasts.io/EpiNow2/dev/articles/estimate_infections_options.html#projecting-the-reproduction-number-with-the-gaussian-process so we'd have to adapt the message depending on what is used.
